src/mmw/js/src/main.js

```diff
@@ -29,6 +29,19 @@ App.on('start', function() {
 
 App.start();
 
+window.numericSort = function(_x, _y) {
```

Comment:
I think this is the right place to attach this to window but consider putting the sort function in its own module. Also, consider adding a comment why this needs to be attached to window.